docs/src/development/submodules.md

```diff
@@ -48,36 +48,36 @@ E: Error validating submodules in tree:
 
 Since the Platform.sh Git server can't connect to GitHub via SSH without being granted an SSH key to do so, you shouldn't use an SSH URL: ``git@github.com:...``, but you should use an HTTPS URL instead: ``https://github.com/...``.
 
-## Use of private git repositories
+## Use private git repositories
 
-When using Git submodules that are hosted on private repositories, using the `https` protocol fails with errors like:
+When using Git submodules that are private repositories, URLs with the HTTPS protocol fail with errors such as the following:
 
 ```bash
 GitProtocolError: unexpected http resp 401 for https://bitbucket.org/myusername/mymodule.git/info/refs?service=git-upload-pack
 ```
 
-To fix this, you need to:
+To fix this, follow these steps:
 
-1. Change your `.gitmodules` file from HTTPS syntax to SSH syntax, e.g.
+1. Change your modules declarations to use SSH for URLs.
 
-    from:
+    So your existing declaration might look like this:
```

Comment:
```suggestion
    To do so, change your existing declaration (similar to the following):
```

Reply:
It's generally against our style to treat code blocks as part of an uninterrupted sentence. Each one should be introduced separately and treated separately.

We also try to avoid parentheticals https://developers.google.com/style/parentheses (this is something I personally struggle with)

Reply:
Ok I'll keep it in mind.